### PR TITLE
Changelog for 19.7.3.9

### PR DESCRIPTION
**Pull request by akuzm**
This is for 19.7.3.9, not everything is backported yet.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
+## ClickHouse release 19.7.3.9, 2019-05-30
+
+### New Features
+* Allow setting constraints such as `max_memory_usage` in users.xml.
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Not descriptive enough.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+  [#4931](https://github.com/yandex/ClickHouse/pull/4931) ([Vitaly
+Baranov](https://github.com/vitlibar))
+* Add a second version of the function `groupUniqArray` with an optional
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: is to -> is similar to

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+  `max_size` parameter that limits the size of the resulting array. This
+behavior is to `groupArray(max_size)(x)` function.
+[#5026](https://github.com/yandex/ClickHouse/pull/5026) ([Guillaume
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: .

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,64 @@
...
+Tassery](https://github.com/YiuRULE))
+* For TSVWithNames/CSVWithNames input file formats, column order can now be
+  determined from file header. This is controlled by
+`input_format_with_names_use_header` parameter.
+[#5081](https://github.com/yandex/ClickHouse/pull/5081)
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: .

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,64 @@
...
+([Ivan](https://github.com/abyss7))
+* Data loss on heavy load via KafkaEngine (#4736)
+[#5080](https://github.com/yandex/ClickHouse/pull/5080)
+([Ivan](https://github.com/abyss7))
+
+### Performance Improvements
+* Use radix sort for sorting by single numeric column in `ORDER BY` without
+  `LIMIT`. [#5106](https://github.com/yandex/ClickHouse/pull/5106)
+([alexey-milovidov](https://github.com/alexey-milovidov))
+* Implement radix sort for `ORDER BY`.
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Duplicate of above.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

File: CHANGELOG.md (hunk 4 above)
Comment (akuzm) [reply CHANGELOG.md]: What's wrong here?

**Pull request merged**